### PR TITLE
Feature: Governance and New Withdraw analytics events

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

Task: [APP-975](https://aragonassociation.atlassian.net/browse/APP-975)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/containers/proposalStepper/index.tsx [hunk 1]
```diff
@@ -60,13 +63,38 @@ const ProposalStepper: React.FC<ProposalStepperType> = ({
         wizardTitle={t('newWithdraw.defineProposal.heading')}
         wizardDescription={t('newWithdraw.defineProposal.description')}
         isNextButtonDisabled={!defineProposalIsValid(dirtyFields, errors)}
+        onNextButtonClicked={next => {
+          trackEvent('newProposal_nextBtn_clicked', {
+            dao_address: dao,
+            step: '1_define_proposal',
+            settings: {
+              author_address: address,
+              title: getValues('proposalTitle'),
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/proposalStepper/index.tsx:73]: I think you can do a single `getValues` call to pull in the needed data like so: 
```javascript 
[proposalTitle, summary] = getValues(['proposalTitle', 'proposalSummary'])
```

Also, I'm curious about why we're tracking all the proposal metadata. I'll reach out to Martin

File: packages/web-app/src/containers/proposalStepper/index.tsx [hunk 2]
```diff
@@ -60,13 +63,38 @@ const ProposalStepper: React.FC<ProposalStepperType> = ({
...
         wizardTitle={t('newWithdraw.setupVoting.title')}
         wizardDescription={t('newWithdraw.setupVoting.description')}
         isNextButtonDisabled={!setupVotingIsValid(errors, durationSwitch)}
+        onNextButtonClicked={next => {
+          trackEvent('newProposal_nextBtn_clicked', {
+            dao_address: dao,
+            step: '2_setup_voting',
+            settings: {
+              start: getValues('startUtc'),
+              end: getValues('endUtc'),
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/proposalStepper/index.tsx]: These two are deceptive. I think you might need a few more properties in addition to build out the actual start and end dates, namely `startDate`, `endDate`, `startTime` and `endTime`. Take a look over [here](https://github.com/aragon/apps/blob/68a705d9b4328fc252878a7c0a986935dce9c5b9/packages/web-app/src/context/createProposal.tsx#L173)

**Review by Fabricevladimir — COMMENTED** [review]
Thanks Rakesh! Left a couple of comments

File: packages/web-app/src/containers/proposalStepper/index.tsx (hunk 2 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/proposalStepper/index.tsx]: Thanks fixed it

File: packages/web-app/src/containers/proposalStepper/index.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/proposalStepper/index.tsx:73]: I haven't seen this before. To use this destructuring pattern, we would have to initialize all values in the first line of the function, which after prettier runs, comes to around 10 lines of code(just the destructured initialization). Also, I believe `getValues` operation would be O(1), so didn't bother changing it here. But I've used this for the next tracking event..

**Review by sepehr2github — APPROVED** [review]

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**